server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -371,6 +373,7 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
             storage,
             appTenants,
             operationsDbMongoManager,
+            new DeltaService(operationsDbMongoManager, tenantManager),
```

Comment:
nit: can we declare this above (like all the other params) rather than doing it in-line?

Reply:
addressed in 51bd2aa